sql/moz-fx-data-shared-prod/fenix_derived/meta_attribution_country_counts_v1/query.sql

```diff
@@ -1,25 +1,54 @@
+{% if is_init() %}
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    CASE
+      WHEN (metrics.string.meta_attribution_app = '382348575493443')
+        THEN 'Firefox Focus for Android'
+      WHEN (metrics.string.meta_attribution_app = '979253712091854')
+        THEN 'Firefox Browser for Android'
+      WHEN (metrics.string.meta_attribution_app = '697946762208244')
+        THEN 'Firefox Nightly for Android'
+      ELSE NULL
+    END AS meta_attribution_app,
+    normalized_channel,
+    metadata.geo.country AS country,
+    COUNT(*) AS ping_count
+  FROM
+    fenix.first_session
+  WHERE
+    DATE(submission_timestamp) >= '2023-11-01'
+    AND client_info.client_id IS NOT NULL
+  GROUP BY
+    submission_date,
+    meta_attribution_app,
+    normalized_channel,
+    country;
+
+{% else %}
 -- Aggregate meta attribution counts by country and channel.
-SELECT
-  DATE(submission_timestamp) AS submission_date,
-  CASE
-    WHEN (metrics.string.meta_attribution_app = '382348575493443')
-      THEN 'Firefox Focus for Android'
-    WHEN (metrics.string.meta_attribution_app = '979253712091854')
-      THEN 'Firefox Browser for Android'
-    WHEN (metrics.string.meta_attribution_app = '697946762208244')
-      THEN 'Firefox Nightly for Android'
-    ELSE CAST(NULL AS STRING)
-  END AS meta_attribution_app,
-  normalized_channel,
-  metadata.geo.country AS country,
-  COUNT(*) AS ping_count
-FROM
-  fenix.first_session
-WHERE
-  DATE(submission_timestamp) = @submission_date
-  AND client_info.client_id IS NOT NULL
-GROUP BY
-  submission_date,
-  meta_attribution_app,
-  normalized_channel,
-  country;
+  SELECT
```

Comment:
Is it standard convention to indent within the jinja blocks?  It feels strange to do since the output will be indented.  I don't really care either way, just curious

Reply:
Yes, the indentation is intended (https://github.com/mozilla/bigquery-etl/commit/a50e36ddc87fbaa5a690aaa3674a6faa52409e12).
In most cases where the raw SQL is used the indentation should go away, since bqetl will reformat the SQL as part of the rendering (https://github.com/mozilla/bigquery-etl/blob/b853dc9ea682881c83352127ea3735676ca65ced/bigquery_etl/util/common.py#L135)